.gitignore

```diff
@@ -0,0 +1,31 @@
+.DS_Store
+*.gem
+*.rbc
+.bundle
+.config
+gems
+coverage
+InstalledFiles
+lib/bundler/man
+pkg
+rdoc
+spec/reports
+test/tmp
+test/version_tmp
+tmp
+
+# YARD artifacts
+.yardoc
+_yardoc
+doc/
+
+*notes.*
+*scratch.*
+todo.md
+
+.rvmrc
+.ruby-gemset
+.ruby-version
+Gemfile.lock
+
+*.log
```

Comment:
This `.gitignore` file doesn't seem like it's the one we use for all our other projects. See: https://github.com/eventide-project/messaging/blob/master/.gitignore